common/display/src/main/scala/uk/ac/wellcome/display/models/v2/DisplayLocationV2.scala

```diff
@@ -20,20 +20,10 @@ sealed trait DisplayLocationV2
 
 object DisplayLocationV2 {
   def apply(location: Location): DisplayLocationV2 = location match {
-    case DigitalLocation(url, locType, license, credit, accessConditions, _) =>
-      DisplayDigitalLocationV2(
-        locationType = DisplayLocationType(locType),
-        url = url,
-        credit = credit,
-        license = license.map(DisplayLicenseV2(_)),
-        accessConditions = accessConditions.map(DisplayAccessCondition(_))
-      )
-    case PhysicalLocation(locationType, label, accessConditions, _) =>
-      DisplayPhysicalLocationV2(
-        locationType = DisplayLocationType(locationType),
-        label = label,
-        accessConditions = accessConditions.map(DisplayAccessCondition(_))
-      )
+    case digitalLocation: DigitalLocation =>
+      DisplayDigitalLocationV2(digitalLocation)
+    case physicalLocation: PhysicalLocation =>
+      DisplayPhysicalLocationV2(physicalLocation)
```

Comment:
Maybe worth removing the V2s here as well?

Reply:
Ha, yes, I was scared of doing the models stuff as it was a bit more complex and might be a long thread to pull... I'll give it a go in a separate PR after this?

Reply:
:+1: np another PR sounds great